docs/source/contributing/submission_guidelines.rst

```diff
@@ -18,6 +18,17 @@ Our efforts will be focused on the 2.x version of DKAN. If you are reporting a b
 and it is not a security issue, we encourage you to reach out on the
 `DKAN Discussions <https://github.com/GetDKAN/dkan/discussions>`_.
 
+Setting up a local development sandbox
+--------------------------------------
+
+We recomend using DDEV with the DDEV-DKAN add-on as there are many helpful commands that will make it easier to get started.
+Once you have composer, docker, and DDEV `installed <https://dkan.readthedocs.io/en/latest/installation/index.html>`_, head over to `Getting Started with DDEV-DKAN <https://getdkan.github.io/ddev-dkan/getting-started.html>`_
```

Comment:
Period at the end of this for consistencies sake. "Getting started with DDEV-DKAN."